collection.go

```diff
@@ -18,6 +18,12 @@ import (
 type CollectionOptions struct {
 	Maps     MapOptions
 	Programs ProgramOptions
+
+	// OverrideMaps defines a set of maps that will be used instead
+	// of creating new ones when loading the object. The maps' specs
+	// should be compatible and there must exist a map in
+	// CollectionSpec.Maps for each map in OverrideMaps.
+	OverrideMaps map[string]*Map
```

Comment:
Bikeshed: `MapOverrides`, `MapReplacements`?

Reply:
No idea. I had named it before `ReplaceMaps` but then realized it was not a very good one. I'll let up to you folks.

Reply:
`MapReplacements` gmv :+1: